ndcube/ndcube.py

```diff
@@ -408,46 +412,47 @@ def axis_world_coords(self, *axes, edges=False, wcs=None):
         return tuple(axes_coords[i] for i in object_indices)
 
     @utils.misc.sanitise_wcs
-    def axis_world_coords_values(self, *axes, edges=False, wcs=None):
+    def axis_world_coords_values(self, *axes, pixel_corners=False, wcs=None):
         """
         Returns WCS coordinate values of all pixels for desired axes.
 
         Parameters
         ----------
-        axes: `int` or `str`, or multiple `int` or `str`
+        axes: `int` or `str`, or multiple `int` or `str`, optional
             Axis number in numpy ordering or unique substring of
-            `~ndcube.NDCube.wcs.world_axis_physical_types`
+            `~ndcube.NDCube.world_axis_physical_types`
             of axes for which real world coordinates are desired.
             axes=None implies all axes will be returned.
 
-        edges: `bool`
-            If True, the coords at the edges of the pixels are returned
-            rather than the coords at the center of the pixels.
-            Note that there are n+1 edges for n pixels which is reflected
-            in the returned coords.
-            Default=False, i.e. pixel centers are returned.
+        pixel_corners: `bool`, optional
+            If `True` then instead of returning the coordinates of the pixel
+            centers the coordinates of the pixel corners will be returned, this
+            increases the size of the output by 1 as all corners are returned.
```

Comment:
```suggestion
            If `True` then instead of returning the coordinates of the pixel
            centers the coordinates of the pixel corners will be returned.  This
            increases the size of the output along each dimension by 1 as all corners are returned.
```